### ISSUE TITLE
Founding cities in WB

### ISSUE THREAD

**Issue by Murderologist** [LoneGazebo/Community-Patch-DLL#11044]
On the latest version. Just using 4.10 no other mods.

When creating a scenario in world builder on 4.10 the game loads just fine with 22 majors and 41 minors. If I begin to set up scenario by founding cities in world builder, the game CTDs immediately upon loading the scenario ingame. I’ve taken a break since 4.2.7. I tested that build and it works fine still. Any help with this would be appreciated.

**Comment by Murderologist** [comment]
^^^

**Comment by artemigkh** [comment]
I had a problem the other day which I think may have the same root cause - loading a custom scenario in general (easily reproducible with loading [Play The World Extended (Brave New World Edition)](https://steamcommunity.com/sharedfiles/filedetails/?id=160163330) as a scenario, but can still play the map normally with random spawns) seems to have been broken recently, because I've played PTWE on earlier versions of VP.

**Comment by skodkim** [comment]
I agree that loading scenario maps seems to be broken in 4.15. A real shame since I always use that.

**Comment by artemigkh** [comment]
If it helps, the exact moment that this broke was https://github.com/LoneGazebo/Community-Patch-DLL/releases/tag/Release-3.8

3.7.12 - can load worldbuilder scenarios
3.8 - crashes on loading worldbuilder scenarios

**Comment by Murderologist** [comment]
Mine stopped work in v124 I think it’s 4.2 but super early 4. Something ever since no joy

**Comment by Murderologist** [comment]
Still nothing

**Comment by axatin** [comment]
I think this is fixed now? @Murderologist can you check?
